.github/workflows/call-pr-3-bump-tag.yml

```
@@ -0,0 +1,19 @@
+# This workflow is used to convert the `.version` in the `metadata.yaml` file
+# into a valid `git tag` on push to `main`.
+# We use the `.version` field in that file to denote the version of the config
+# once a PR is merged.
+name: Call pr-3-bump-tag.yml
+on:
+  push:
+    branches:
+      - 'release-*'
+    paths:
+      - 'metadata.yaml'
+jobs:
+  call:
+    # We simply call the workflow on main so we don't have to propagate CI changes to
+    # multiple config branches all the time.
+    uses: codegat-test-org/configs/.github/workflows/pr-3-bump-tag.yml@main
```

```suggestion
    uses: access-nri/access-om2-configs/.github/workflows/pr-3-bump-tag.yml@main
```

Still pointing to your test org. Better check my suggestion is correct.